spec/Layers/FeatureLayer/FeatureManagerSpec.js

```diff
@@ -756,6 +756,54 @@ describe('L.esri.FeatureManager', function () {
     server.respond();
   });
 
+  it('should wrap the updateFeatures method on the underlying service and refresh', function (done) {
+    server.respondWith('POST', 'http://gis.example.com/mock/arcgis/rest/services/MockService/MockFeatureServer/0/updateFeatures', JSON.stringify({
+      'updateResults': [{
+        'objectid': 1,
+        'success': true
+      }, {
+        'objectid': 2,
+        'success': true
+      }]
+    }));
+
+    layer.updateFeatures({
+      type: 'FeatureCollection',
+      features: [{
+        type: 'Feature',
+        id: 1,
+        properties: {
+          foo: 'bar'
+        },
+        geometry: {
+          type: 'Point',
+          coordinates: [-121, 45]
+        }
+      }, {
+        type: 'Feature',
+        id: 2,
+        properties: {
+          foo: 'bar'
+        },
+        geometry: {
+          type: 'Point',
+          coordinates: [-121, 45]
+        }
+      }]
+    }, function (error, response) {
+      expect(response).to.deep.equal([{
```

Comment:
i noticed that the syntax errors i fixed in 872de35 didn't actually induce an error in your test because you're currently only introspecting the mock service response (and not the actual _request_ parameters) which this lib actually has control over.